x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -346,8 +384,74 @@ export const EndpointList = () => {
           );
         },
       },
+      {
+        field: '',
+        name: i18n.translate('xpack.securitySolution.endpoint.list.actions', {
+          defaultMessage: 'Actions',
+        }),
+        actions: [
+          {
+            // eslint-disable-next-line react/display-name
+            render: (item: HostInfo) => {
+              return (
+                <TableRowActions
+                  items={[
+                    <EuiContextMenuItem icon="logoSecurity" key="hostDetailsLink">
+                      <LinkToApp
+                        data-test-subj="hostLink"
+                        appId="securitySolution"
+                        appPath={`hosts/${item.metadata.host.hostname}`}
+                        href={`${services?.application?.getUrlForApp('securitySolution')}/hosts/${
```

Comment:
For the Security Solution URLs, I think methods exists (similar to the ones we have) that return both the `path` and the `href`. We should use those instead of building these paths manually